### PR TITLE
DevWs for CoolKit Technology Co.,Ltd containing containing ZigBee Switch

### PR DESCRIPTION
**Pull request by mingwei0827**

### CODE REVIEW

**Review by dsobierajsk — APPROVED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -102,6 +102,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "1000", manufacturer: "SONOFF", model: "01MINIZB", deviceJoinName: "SONOFF 01MINIZB" //01MINIZB
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "S26R2ZB", deviceJoinName: "SONOFF Plug", ocfDeviceType: "oic.d.smartplug" //SONOFF S26R2 Plug
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "S40LITE", deviceJoinName: "SONOFF Plug", ocfDeviceType: "oic.d.smartplug" //SONOFF S40Lite Plug
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0020, FC57", outClusters: "0019", manufacturer: "SONOFF", model: "ZBMINI-L", deviceJoinName: "SONOFF ZBMINI-L" //SONOFF ZBMINI-L
```
Comment (dsobierajsk) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please change deviceJoinName to "SONOFF Plug"

**Review by dsobierajsk — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 1 above)
Comment (k-morawski) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Dear @mingwei0827 , could you please check Dawid's comment? Many thanks!

Comment (dsobierajsk) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Dear @mingwei0827, if it is not a plug, please change deviceJoinName to "SONOFF Switch" according to naming convention seen in every fingerprint

**Comment by mingwei0827** [comment]
@dsobierajsk hi~I have corrected it. Please help to check whether it is OK, thank you~

**Comment by mingwei0827** [comment]
@dsobierajsk hi，I have corrected it. Please help to check whether it is OK, thank you。

**Review by dsobierajsk — APPROVED** [review]

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**